source/viz.md

```diff
@@ -920,16 +974,30 @@ further improve the clarity and accessibility of your visualization.
 
 ```{code-cell} ipython3
```

Comment:
The note box right above this code doesn't need to be a note box. Just write in the text that we are using (switching back to??) mark point and specifying shape attr